instance_skel.js

```diff
@@ -432,4 +432,8 @@ instance.extendedBy = function (module) {
 	util.inherits(module, instance)
 }
 
+instance.handleHttpRequest = function () {
```

Comment:
I would prefer this to not have a default implementation. You are already checking if this function exists before calling it, so dropping this should have no change.
And with a default implementation companion wont be able to tell what modules have implementations and what are using the base implementation

Reply:
Sure, this can be changed. I only added it when the PR Checks errored as I originally had it as an abstract type and so didn't pass the checks when it encountered a module that didn't define it (ie, all of them). I've already removed the abstract keyword so I should be able to remove that default function and still pass the PR checks.